scripts/InstallRunNodeMapnikServer.sh

```diff
@@ -0,0 +1,23 @@
+# Install and run node-mapnik-server
+if [ -z "$NODE_MAPNIK_SERVER_PORT" ]; then
+    echo "Error: NODE_MAPNIK_SERVER_PORT not set."
+    false
+else
+    rm -rf ~/tmp/npm*
+    cd $HOOT_HOME
+    source SetupEnv.sh
+    source conf/DatabaseConfig.sh
+    cd node-mapnik-server
+    npm install -q
+    NPID=`ps aux | grep node | grep "app.js hoot_style.xml $NODE_MAPNIK_SERVER_PORT" | awk '{print $2}'`
+    # send kill
+    if [ -z "$NPID" ]; then
+        #Do nothing
+        :
+    else
+        kill $NPID;
+    fi
+    mkdir -p log
+    node app.js hoot_style.xml $NODE_MAPNIK_SERVER_PORT 1>log/app.log 2>&1 &
+fi
+
```

Comment:
On second thought, going to leave InstallRunNodeMapnikServer.sh in place.  The vagrant install specifies the ports to use for the different application in the vagrantfile, whereas selma uses separate logic in its hoot-release/conf/Config.sh to determine what ports to use.  Since InstallRunNodeMapnikServer.sh is set up to look for NODE_MAPNIK_SERVER_PORT to support how selma works, it needs to stay around I think.